docs/modules/ROOT/nav-how-to-guides.adoc

```diff
@@ -13,6 +13,7 @@
 *** xref:how-to-guides/testing_applications/surface-level_tests.adoc[Surface-level tests]
 *** xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[Adding an integration test]
 *** xref:how-to-guides/testing_applications/proc_creating_custom_test.adoc[Creating a custom integration test]
+*** xref:pages/how-to-guides/testing_applications/proc_retriggering_integration_tests.adoc[Retriggering integration tests]
```

Comment:
See how the links above this all just link to `xref:how-to-guides/...`? I think you need to nix that `pages/` prefix:

```suggestion
*** xref:how-to-guides/testing_applications/proc_retriggering_integration_tests.adoc[Retriggering integration tests]
```

Reply:
Oh, actually - I just checked the demo link and your change actually works as written. I don't know why!

Reply:
That's interesting, seems like `pages` is required for links to work in github but not in the site itself.  I'll revert that change.